internal/entitlement/connector.go

```diff
@@ -20,6 +20,7 @@ const (
 type ListEntitlementsParams struct {
 	Namespace      string
 	SubjectKey     string
+	FeatureIDs     []string
```

Comment:
lets call it Keys if it filters for keys

Reply:
Nice catch. Will open a PR with fix. Actually we need to filter based on _ids_ not _keys_.